README.md

```diff
@@ -1,13 +1,87 @@
 [![GitHub Discussions](https://img.shields.io/github/discussions/keptn-sandbox/lifecycle-controller)](https://github.com/keptn-sandbox/lifecycle-controller/discussions)
 
 The purpose of this repository is to demonstrate and experiment with
 a prototype of a _**Keptn Lifecycle Controller**_.
 The goal of this prototype is to introduce a more “cloud-native” approach for pre- and post-deployment, as well as the concept of application health checks.
 It is an experimental project, under the umbrella of the [Keptn Application Lifecycle working group](https://github.com/keptn/wg-app-lifecycle).
 
+## Deploy the latest release
+
+The lifecycle controllers includes a Mutating Webhook which requires TLS certificates to be mounted as a volume in its pod. The certificate creation
+is handled automatically by [cert-manager](https://cert-manager.io). To install **cert-manager**, follow their [installation instructions](https://cert-manager.io/docs/installation/).
+
+When the *cert-manager* is installed, you can run
+
+<!---x-release-please-start-version-->
+
+```
+kubectl apply -f https://github.com/keptn-sandbox/lifecycle-controller/releases/download/v0.1.0/release.yaml
+```
+
+<!---x-release-please-end-->
+
+to install the latest release of the lifecycle controller.
+
+## Goals
+
+The Keptn Lifecycle Controller aims at supporting Cloud Native teams with:
+
+- Pre-requisite evaluation before deploying Workloads and applications
+- Finding out when an application (not workload) is ready and working
+- Checking the Application Health in a declarative (cloud-native) way
+- Standardized way for pre- and post-deployment tasks
+- Provide out-of-the-box Observability of the deployment cycle
+
+![](./assets/operator-maturity.jpg)
+
+The Keptn Lifecycle Controller could be seen as a general purpose and declarative Level 3 operator for your Application.
+For this reason, the Keptn Lifecycle Controller is agnostic to the Deployment tool that is being used and works with any GitOps solution.
+
+## How to use
+
+The Keptn Lifecycle Controller monitors manifests that have been applied against the KubeAPI and reacts if it finds a workload with special annotations.
+For this, you should annotate your [Workload](https://kubernetes.io/docs/concepts/workloads/) with (at least) the following two annotations:
+
+```yaml
+keptn.sh/app: myAwesomeAppName
+keptn.sh/workload: myAwesomeWorkload
+```
+
+In the case you want to run pre- and post-deployment checks, further annotations are necessary:
+
+```yaml
+keptn.sh/pre-deployment-tasks: verify-infrastructure-problems
+keptn.sh/post-deployment-tasks: slack-notification,performance-test
+```
+
+In this example, before the deployment starts, a check for open problems in your infrastructure are performed. If everything is fine, the deployment continues and afterward, a slack notification is sent with the result of the deployment and a pipeline to run performance tests is invoked.
```

Comment:
Is there a function for that or is this only an example? If this is only an example, we should write this there ...